.docker/opt/configure-aiida.sh

```diff
@@ -76,5 +76,11 @@ verdi daemon stop
 # Migration will run for the default profile.
 verdi storage migrate --force || echo "Database migration failed."
 
+export ARCH=`uname -m`
```

Comment:
Please add a small comment here that explains why we are doing this and when can it be removed.

Reply:
Thanks! I forgot to push the last commit. It is now added.